webpack/assets/javascripts/react_app/common/urlHelpers.js

```diff
@@ -28,7 +28,7 @@ export const getURIpage = () => Number(getURI().query(true).page) || 1;
 /**
  * Get updated perPage param
  */
-export const getURIperPage = () => Number(getURI().query(true).per_page);
+export const getURIperPage = () => Number(getURI().query(true).per_page) || 25;
```

Comment:
We have the `entries_per_page` setting. Perhaps this should honor it? It also defaults to 20 so if you don't honor it, matching defaults might be a good thing.

Reply:
Don't we have this metadata now as part of the app component?